sections/content_plots/case_evolution.R

```diff
@@ -1,5 +1,18 @@
+output$selectize_evolByCanton <- renderUI({
+  selectizeInput(
+    "selectize_evolByCanton",
+    label    = "Select Kanton",
+    choices  = c("Alle", unique(data_evolution$name))[order(unique(data_evolution$name))],
+    selected = "Alle",
+    multiple = FALSE
+  )
+})
+
 output$case_evolution <- renderPlotly({
+  req(input$selectize_evolByCanton)
+  
   data <- data_evolution %>%
+    filter(if (identical(input$selectize_evolByCanton, "Alle")) TRUE else name %in% input$selectize_evolByCanton) %>%
```

Comment:
What's the advantage of using identical() over ` input$selectize_evolByCanton == "Alle"`. I don't know the function

Reply:
it returns not a vector but a true/false value which helps in if clauses. you can think of it as `all(input$selectize_evolByCanton == "Alle")`

Reply:
That's cool! Thanks for the input :) didn't know about that